packages/common/src/api/tan-query/useTrackFileInfo.ts

```diff
@@ -0,0 +1,36 @@
+import { BlobInfo, Id } from '@audius/sdk'
+import { useQuery } from '@tanstack/react-query'
+
+import { useAudiusQueryContext } from '~/audius-query'
+import { ID } from '~/models/Identifiers'
+
+import { QUERY_KEYS } from './queryKeys'
+import { QueryKey, SelectableQueryOptions } from './types'
+
+export const getTrackFileInfoQueryKey = (trackId: ID | null | undefined) => {
+  return [QUERY_KEYS.trackFileInfo, trackId] as unknown as QueryKey<BlobInfo>
+}
+
+export const useTrackFileInfo = <TResult = BlobInfo>(
```

Comment:
Haven't written one of these in a while and I'm not sure what the canonical pattern is anymore. Hopefully I got close enough!

Reply:
i call upon @DejayJD